lbrynet/conf.py

```diff
@@ -268,7 +275,7 @@ def _convert_value(value):
     'min_info_rate': (float, .02),  # points/1000 infos
     'min_valuable_hash_rate': (float, .05),  # points/1000 infos
     'min_valuable_info_rate': (float, .05),  # points/1000 infos
-    'peer_port': (int, 3333),
+    'peer_port' : (int, DEFAULT_PEER_PORT),
```

Comment:
Extraneous space between key string and colon.

Reply:
ops, thanks (for both)!